### DISCUSSION TITLE
Which is best Hardhat or Truffle?

### DISCUSSION THREAD

**Discussion by steven9255** [FreshAirCoin/SmartContract#2] [Q&A] [answered]
Hello. @cryptoguy55 
Which is best Hardhat or Truffle?
Could you explain difference of the hardhat and truffle?
Thank you Steven

**Comment by stackcraft55** [comment, marked as answer]
Hello. Steven 
When it comes to unit tests, there are two Hardhat plugins that support the Truffle testing APIs: hardhat-truffle4 and hardhat-truffle5 . Using these you can run your existing tests with Hardhat
Please read this articles
It will help your understanding
Thank you
